### DISCUSSION TITLE
can't use it in vue3

### DISCUSSION THREAD

**Discussion by robertnicjoo** [biati-digital/glightbox#466] [Q&A]
I am trying to use this in vuejs 3 as follow:

```js
import { createSSRApp } from 'vue'
import GLightbox from 'glightbox';

const app = createSSRApp(App)
app.use(GLightbox())
```

And I get `Uncaught ReferenceError: GLightbox is not defined`

Is there something I'm doing wrong?

**Comment by gingerchew** [comment]
GLightbox is a client side script, using it in a server side app without a wrapper/plugin of some kind isn't possible. If you can make a codepen/sandbox we can try and work something out.
